Signal/src/ViewControllers/SignalsViewController.m

```diff
@@ -414,7 +421,24 @@ - (void)updateShouldObserveDBModifications
 
 - (void)setShouldObserveDBModifications:(BOOL)shouldObserveDBModifications
 {
-    if (!_shouldObserveDBModifications && shouldObserveDBModifications && self.threadMappings != nil) {
```

Comment:
nit: Can we rename this to reflect it's new behavior? Maybe: `shouldHandleDBModifications`